### PR TITLE
chromeCastAutomationHook

### PR DESCRIPTION
**Pull request by bindumadhav**

### CODE REVIEW

File: ChromecastSampleApp/app/build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ android {
     defaultConfig {
         applicationId "com.ooyala.sample.ChromecastSampleApp"
         minSdkVersion 14
-        targetSdkVersion 23
+        targetSdkVersion 21
```
Comment (lental) [inline ChromecastSampleApp/app/build.gradle]: do you know why you did this?

Comment (lental) [reply ChromecastSampleApp/app/build.gradle]: bump

Comment (bindumadhav) [reply ChromecastSampleApp/app/build.gradle]: oops, got pulled in diff directions , will get back with soon with the answer .

Comment (bindumadhav) [reply ChromecastSampleApp/app/build.gradle]: I was running into compilation error in one of the mac minis( some combination of Android build tools and targetSdkVersion ) was failing . 
I tried again now with targetSdkVersion 23 and it works just fine . 
Shall i change it back to 23 ?

Shall i change the targetSdkVersion back to 23 ?

Comment (bindumadhav) [reply ChromecastSampleApp/app/build.gradle]: reverted targetSdkVersion back to 23 .

**Pull request merged**